# Icelandic translation for update-manager
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2008-04-02 10:42+0200\n"
"PO-Revision-Date: 2008-03-21 19:29+0000\n"
"Last-Translator: Tómas Árnason <arnason@samlokan.sytes.net>\n"
"Language-Team: Icelandic <is@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2008-04-02 05:49+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:83 ../UpdateManager/Common/utils.py:83
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:86 ../UpdateManager/Common/utils.py:86
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:89 ../UpdateManager/Common/utils.py:89
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:92 ../UpdateManager/Common/utils.py:92
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:194 ../DistUpgrade/distro.py:401
#, python-format
msgid "Server for %s"
msgstr "Netþjónn fyrir %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:213 ../DistUpgrade/distro.py:218
#: ../DistUpgrade/distro.py:232
msgid "Main server"
msgstr "Aðalnetþjónn"

#: ../DistUpgrade/distro.py:235
msgid "Custom servers"
msgstr "Sérsniðnir netþjónar"

#: ../DistUpgrade/dist-upgrade.py:15
#, fuzzy
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Notið gefna slóð til að leita á geisladrifi að uppfæranlegum pökkum"

#: ../DistUpgrade/dist-upgrade.py:21
#, fuzzy
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Veljið viðmót. Þau sem í boði eru:\n"
"Texta-viðmót, Gtk-viðmót, KDE-viðmót"

#: ../DistUpgrade/dist-upgrade.py:24
#, fuzzy
msgid ""
"Use special upgrade mode. Available:\n"
"desktop, server"
msgstr ""
"Veljið uppfærsluham. Þeir sem í boði eru:\n"
"Neytandaútgáfa, Netþjónsútgáfa"

#: ../DistUpgrade/DistUpgradeCache.py:73
#, fuzzy
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjarlægið pakka í slæmu ástandi"
msgstr[1] "Fjarlægið pakka í slæmu ástandi"

#: ../DistUpgrade/DistUpgradeCache.py:76
#, fuzzy, python-format
msgid ""
"The package '%s' is in a inconsistent state and needs to be reinstalled, but "
"no archive can be found for it. Do you want to remove this package now to "
"continue?"
msgid_plural ""
"The packages '%s' is in a inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgstr[0] ""
"Hugbúnaðarpakkinn '%s' er í einhverju rugli og það þarf að enduruppsetja "
"hann, en ekki er hægt að finna neinn stað þar sem hann er geymdur. Viltu "
"frjarlægja þennan pakka áður en haldið er áfram?"
msgstr[1] ""
"Hugbúnaðarpakkarnir '%s' er í einhverju rugli og það þarf að enduruppsetja "
"þá, en ekki er hægt að finna neina staði þar sem þeir eru geymdir. Viltu "
"frjarlægja þessa pakka áður en haldið er áfram?"

#: ../DistUpgrade/DistUpgradeCache.py:241
msgid "Broken packages"
msgstr "Bilaðir pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:242
#, fuzzy
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Kerfið þitt inniheldur bilaða pakka sem ekki var hægt að laga með þessum "
"hugbúnaði. Vinsamlegast leystu vandann með synaptic eða apt-get áður en "
"lengra er haldið."

#: ../DistUpgrade/DistUpgradeCache.py:570
msgid "Can't upgrade required meta-packages"
msgstr "Get ekki uppfært nauðsynlega lýsi-pakka"

#: ../DistUpgrade/DistUpgradeCache.py:574
#, fuzzy
msgid "A essential package would have to be removed"
msgstr "Nauðsynlegan pakka þyrfti að fjarlægja"

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:581
#, fuzzy
msgid ""
"A unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Það kom upp óleysanlegt vandamál við útreikning á uppfærslu.\n"
"\n"
" Orsökin gæti hafa verið:\n"
" *Að uppfæra yfir í prufu útgáfu af Ubuntu\n"
" *Að keyra nýjustu prufu útgáfu af Ubuntu\n"
" *Að nota óopinbera pakka sem fylja ekki með Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:591
msgid "This is most likely a transient problem, please try again later."
msgstr "Þetta er væntanlega tímabundið vandamál. Reynið aftur síðar."

#: ../DistUpgrade/DistUpgradeCache.py:594
#, fuzzy
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the "
"bugreport."
msgstr ""
"Ef þetta virkar ekki, þá tilkynnið vinsamlegast um galla gagnvart 'update-"
"manager' pakkanum og látið fylgja skrárnar í /var/log/dist-upgrade/ í "
"villutilkynningunni."

#: ../DistUpgrade/DistUpgradeCache.py:599
#: ../UpdateManager/UpdateManager.py:899
#, fuzzy
msgid "Could not calculate the upgrade"
msgstr "Gat ekki reiknað uppfærsluna"

#. FIXME: maybe ask a question here? instead of failing?
#: ../DistUpgrade/DistUpgradeCache.py:641
msgid "Error authenticating some packages"
msgstr "Villa við sannvottun einhverra pakka"

#: ../DistUpgrade/DistUpgradeCache.py:642
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Ekki tókst að sannvotta einhverja pakka. Þetta gæti verið skammvinnt "
"netvandamál. Þú gætir reynt aftur síðar. Sjá hér fyrir neðan fyrir lista "
"yfir pakka sem tókst að sannvotta."

#: ../DistUpgrade/DistUpgradeCache.py:745
#, python-format
msgid "Can't install '%s'"
msgstr "Get ekki sett upp '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:746
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr ""
"Það reyndist ómögulegt að setja upp umbeðinn pakka. Vinsamlegast tilkynntu "
"þetta sem hugbúnaðarvillu. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:753
msgid "Can't guess meta-package"
msgstr "Get ekki giskað á lýsi-pakka"

#: ../DistUpgrade/DistUpgradeCache.py:754
#, fuzzy
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Kerfi þitt inniheldur ekki ubuntu-desktop, kubuntu-desktop, xubuntu-desktop "
"eða edubuntu-desktop pakkana og það var ekki hægt að skynja hvaða útgáfu af "
"Ubuntu þú ert að keyra.\n"
" Vinsamlegast setjið fyrst upp einn af ofangreindum pökkum með synaptic eða "
"apt-get áður en haldið er áfram."

#: ../DistUpgrade/DistUpgradeController.py:92
msgid "Failed to add the CD"
msgstr "Ekki tókst að bæta við geisladisknum"

#: ../DistUpgrade/DistUpgradeController.py:93
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Það kom babb í bátinn þegar geisladisknum var bætt við, Hætt verður við "
"uppfærsluna. Vinsamlegast tilkynnið þetta sem hugbúnaðarvillu ef þetta er "
"gildur Ubuntu geisladiskur.\n"
"\n"
"Villuskilaboðin voru:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:125
msgid "Reading cache"
msgstr "Les skyndiminni"

#: ../DistUpgrade/DistUpgradeController.py:205
msgid "Unable to get exclusive lock"
msgstr "Fékk ekki útilokunarlæsingu"

#: ../DistUpgrade/DistUpgradeController.py:206
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Venjulega þýðir þetta að annað pakkastjórnunarforrit (t.d. apt-get eða "
"aptitude) er í gangi. VInsamlgast lokaðu því forriti fyrst."

#: ../DistUpgrade/DistUpgradeController.py:226
#, fuzzy
msgid "Continue running under SSH?"
msgstr "Halda áfram að keyra undir SSH?"

#: ../DistUpgrade/DistUpgradeController.py:227
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:242
#, fuzzy
msgid "Starting additional sshd"
msgstr "Ræsi auka sshd"

#: ../DistUpgrade/DistUpgradeController.py:243
#, python-format
msgid ""
"To make recovery in case of failure easier a additional sshd will be started "
"on port '%s'. If anything goes wrong with the running ssh you can still "
"connect to the additional one.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:307
#: ../DistUpgrade/DistUpgradeController.py:330
msgid "Can not upgrade"
msgstr "Get ekki uppfært"

#: ../DistUpgrade/DistUpgradeController.py:308
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:331
#, fuzzy
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Python uppsetning þín er skemmd. Vinsamlegast lagaðu táknræna tengilinn "
"(symlink) í '/usr/bin/python'."

#: ../DistUpgrade/DistUpgradeController.py:349
msgid "Package 'debsig-verify' is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:350
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:360
msgid "Include latest updates from the Internet?"
msgstr "Viltu hafa með nýjustu uppfærslurnar af internetinu?"

#: ../DistUpgrade/DistUpgradeController.py:361
msgid ""
"The upgrade process can automatically download the latest updates and "
"install them during the upgrade.  The upgrade will take longer, but when it "
"is complete, your system will be fully up to date.  You can choose not to do "
"this, but you should install the latest updates soon after upgrading."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:501
msgid "No valid mirror found"
msgstr "Enginn gildur spegill fannst"

#: ../DistUpgrade/DistUpgradeController.py:502
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found.This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'no' the update will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:522
msgid "Generate default sources?"
msgstr "Búa til sjálfgefin frumgögn?"

#: ../DistUpgrade/DistUpgradeController.py:523
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No' the update will "
"cancel."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:557
msgid "Repository information invalid"
msgstr "Ógildar útibúaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:558
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"Þegar útibúaupplýsingarnar voru uppfærðar kom upp villa. Vinsamlegast "
"tilkynntu þetta."

#: ../DistUpgrade/DistUpgradeController.py:564
msgid "Third party sources disabled"
msgstr "Frumgögn frá þriðja aðila voru afvirkjuð."

#: ../DistUpgrade/DistUpgradeController.py:565
#, fuzzy
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Nokkrar færslur frá þriðja aðila í sources.list voru gerðar óvirkar. Þú "
"getur endurvirkjað þær eftir uppfærsluna með hugbúnaðar eiginleika tólinu "
"eða pakkastjóranum þínum."

#: ../DistUpgrade/DistUpgradeController.py:599
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] ""
msgstr[1] ""

#: ../DistUpgrade/DistUpgradeController.py:602
#, python-format
msgid ""
"The package '%s' is in a inconsistent state and needs to be reinstalled, but "
"no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' is in a inconsistent state and needs to be reinstalled, "
"but no archive can be found for it.Please reinstall the packages manually or "
"remove them from the system."
msgstr[0] ""
msgstr[1] ""

#: ../DistUpgrade/DistUpgradeController.py:646
msgid "Error during update"
msgstr "Villa við uppfærslu"

#: ../DistUpgrade/DistUpgradeController.py:647
#, fuzzy
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Vandamál kom upp við uppfærslu. Þetta er yfirleitt einhvers konar "
"netvandamál, vinsamlega athugaðu nettenginguna þína og reyndu aftur."

#: ../DistUpgrade/DistUpgradeController.py:656
msgid "Not enough free disk space"
msgstr "Ekki nægilegt diskapláss laust"

#: ../DistUpgrade/DistUpgradeController.py:657
#, fuzzy, python-format
msgid ""
"The upgrade aborts now. The upgrade needs a total of %s free space on disk '%"
"s'. Please free at least an additional %s of disk space on '%s'. Empty your "
"trash and remove temporary packages of former installations using 'sudo apt-"
"get clean'."
msgstr ""
"Uppfærslan slítur vinnu núna. Uppfærslan þarfnast samtals %s af lausu plássi "
"á diski '%s'. Vinsamlega leystu að minnsta kosti auka %s af diskplássi á '%"
"s'. Tæmdu ruslafötuna þína og eyddu tímabundna pakka af eldri uppsetningum "
"með að nota skipunina 'sudo apt-get clean'."

#: ../DistUpgrade/DistUpgradeController.py:768
#, fuzzy
msgid "Support for some applications ended"
msgstr "Stuðningur fyrir sum forrit er farinn"

#: ../DistUpgrade/DistUpgradeController.py:769
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:792
msgid "Do you want to start the upgrade?"
msgstr "Langar þig að hefja uppfærsluna?"

#: ../DistUpgrade/DistUpgradeController.py:856
#: ../DistUpgrade/DistUpgradeController.py:910
msgid "Could not download the upgrades"
msgstr "Gat ekki halað niður uppfærslunum"

#: ../DistUpgrade/DistUpgradeController.py:857
#: ../DistUpgrade/DistUpgradeController.py:911
msgid ""
"The upgrade aborts now. Please check your internet connection or "
"installation media and try again. "
msgstr ""
"Uppfærslan slítur vinnu núna. Vinsamlega athugaðu nettenginguna þína eða "
"uppsetningarmiðla og reyndu aftur. "

#. invoke the frontend now
#: ../DistUpgrade/DistUpgradeController.py:884
#, fuzzy
msgid ""
"The upgrade aborts now. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Uppfærslan slítur vinnu núna. Kerfið þitt gæti verið í óstöðugu ástandi. "
"Endurrétting mun keyrast núna (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:889
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bugreport.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:893
msgid "Could not install the upgrades"
msgstr "Gat ekki sett upp uppfærslurnar"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid "Remove obsolete packages?"
msgstr "Fjarlægja úrelta pakka?"

#: ../DistUpgrade/DistUpgradeController.py:1017
#: ../DistUpgrade/DistUpgrade.glade.h:20
msgid "_Keep"
msgstr "_Halda"

#: ../DistUpgrade/DistUpgradeController.py:1017
msgid "_Remove"
msgstr "_Fjarlægja"

#: ../DistUpgrade/DistUpgradeController.py:1028
msgid "Error during commit"
msgstr "Villa við staðfestingu"

#: ../DistUpgrade/DistUpgradeController.py:1029
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1156
msgid "Restoring original system state"
msgstr "Endurstilli stýrikerfi í upprunalegt form"

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1194
msgid "Required depends is not installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1195
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr ""

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1417
#: ../DistUpgrade/DistUpgradeController.py:1479
msgid "Checking package manager"
msgstr "Athuga pakkastjóra"

#: ../DistUpgrade/DistUpgradeController.py:1422
#, fuzzy
msgid "Preparing the upgrade failed"
msgstr "Uppfærslu undirbúningur mistókst"

#: ../DistUpgrade/DistUpgradeController.py:1423
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bugreport."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1438
msgid "Getting upgrade prerequisites failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1439
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bugreport."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1465
msgid "Updating repository information"
msgstr "Uppfæri útibúa upplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:1490
msgid "Invalid package information"
msgstr "Ógildar pakkaupplýsingar"

#: ../DistUpgrade/DistUpgradeController.py:1491
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the "
"bugreport."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1502
#, fuzzy
msgid "Calculating the changes"
msgstr "Sæki og set upp uppfærslur"

#: ../DistUpgrade/DistUpgradeController.py:1508 ../update-manager:96
#, fuzzy
msgid "Fetching"
msgstr "Að sækja"

#: ../DistUpgrade/DistUpgradeController.py:1518 ../update-manager:100
msgid "Upgrading"
msgstr "Uppfæri"

#: ../DistUpgrade/DistUpgradeController.py:1527
msgid "Searching for obsolete software"
msgstr "Leita að úreltum hugbúnaði"

#: ../DistUpgrade/DistUpgradeController.py:1532
msgid "System upgrade is complete."
msgstr "Uppfærslu á stýrikerfi lokið"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:108
#: ../DistUpgrade/DistUpgradeViewKDE.py:167
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Vinsamlega settu '%s' í drif '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:127
#: ../DistUpgrade/DistUpgradeViewKDE.py:180
msgid "Fetching is complete"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:139
#: ../DistUpgrade/DistUpgradeViewKDE.py:193
#, python-format
msgid "Fetching file %li of %li at %sb/s"
msgstr "Sæki skrá %li af %li hjá %sb/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:140
#: ../DistUpgrade/DistUpgradeViewGtk.py:271
#: ../DistUpgrade/DistUpgradeViewKDE.py:194
#: ../DistUpgrade/DistUpgradeViewKDE.py:327
#, fuzzy, python-format
msgid "About %s remaining"
msgstr "Um %s eftir"

#: ../DistUpgrade/DistUpgradeViewGtk.py:142
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#, python-format
msgid "Fetching file %li of %li"
msgstr "Sæki skrá %li af %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:175
#: ../DistUpgrade/DistUpgradeViewKDE.py:231
msgid "Applying changes"
msgstr "Sett inn breytingar"

#: ../DistUpgrade/DistUpgradeViewGtk.py:200
#: ../DistUpgrade/DistUpgradeViewKDE.py:243
#, python-format
msgid "Could not install '%s'"
msgstr "Gat ekki sett upp '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:201
#: ../DistUpgrade/DistUpgradeViewKDE.py:244
#, fuzzy, python-format
msgid ""
"The upgrade will continue but the '%s' package may be in a not working "
"state. Please consider submitting a bugreport about it."
msgstr ""
"Uppfærslan mun halda áfram en pakkinn '%s' gæti óvirkur. Vinsamlega íhugaðu "
"að senda villuskýrslu um þetta."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:263
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Þú munt tapa öllum breytingum sem þú hefur gert við þessa samskipunarskrá ef "
"þú ákveður að skipta henni út með nýrri útgáfu."

#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewKDE.py:280
msgid "The 'diff' command was not found"
msgstr "'diff' skipunin fannst ekki"

#: ../DistUpgrade/DistUpgradeViewGtk.py:409
#: ../DistUpgrade/DistUpgradeViewText.py:98
msgid "A fatal error occured"
msgstr "Það kom upp grafalvarleg villa"

#: ../DistUpgrade/DistUpgradeViewGtk.py:410
#, fuzzy
msgid ""
"Please report this as a bug (if you haven't already) and include thefiles /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade aborts now.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Vinsamlega tilkynntu þetta sem villu (ef þú hefur ekki gert það nú þegar) og "
"hafðu skrárnar  /var/log/dist-upgrade/main.log og /var/log/dist-upgrade/apt."
"log innifaldar í skýrslunni. Uppfærslan hættir vinnu núna.\n"
"Upprunalega sources.list var vistuð í /etc/apt/sources.list.distUpgrade."

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:539
msgid "To prevent data loss close all open applications and documents."
msgstr "Til að fyrirbyggja tap á gögnum skaltu loka öllum forritum og skrám"

#: ../DistUpgrade/DistUpgradeViewGtk.py:553
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk.py:555
#: ../DistUpgrade/DistUpgradeViewKDE.py:653
#, python-format
msgid "Remove %s"
msgstr "Fjarlægja %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:560
#: ../DistUpgrade/DistUpgradeViewKDE.py:655
#, python-format
msgid "Install %s"
msgstr "Uppsetning á %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:562
#: ../DistUpgrade/DistUpgradeViewKDE.py:657
#, python-format
msgid "Upgrade %s"
msgstr "Uppfæri %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:168
msgid "Media Change"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:292
msgid "Show Difference >>>"
msgstr "Sýna mismun >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:295
msgid "<<< Hide Difference"
msgstr "<<< Fela mismun"

#: ../DistUpgrade/DistUpgradeViewKDE.py:535
msgid "Show Terminal >>>"
msgstr "Sýna Útstöð >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:538
msgid "<<< Hide Terminal"
msgstr "<<< Fela Útstöð"

#: ../DistUpgrade/DistUpgradeViewKDE.py:637
#: ../DistUpgrade/DistUpgradeViewKDE.py:666
#: ../DistUpgrade/DistUpgradeViewKDE.py:669
#: ../DistUpgrade/DistUpgrade.glade.h:9
msgid "Details"
msgstr "Nánar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:682
#: ../DistUpgrade/DistUpgrade.glade.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Hætta við uppfærslu sem er þegar hafin?</big></b>\n"
"\n"
"Kerfið gæti orðið ónothæft ef þú hættir við uppfærlsuna. Það er sterklega "
"mælt með að þú haldir áfram með uppfærsluna."

#: ../DistUpgrade/DistUpgradeViewKDE.py:686
msgid "Cancel Upgrade?"
msgstr "Hætta við uppfærslu?"

#: ../DistUpgrade/DistUpgradeView.py:34
#, python-format
msgid "%li days %li hours %li minutes"
msgstr "%li dagar %li klukkustundir %li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:36
#, python-format
msgid "%li hours %li minutes"
msgstr "%li klukkustundir %li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:38
#, python-format
msgid "%li minutes"
msgstr "%li mínútur"

#: ../DistUpgrade/DistUpgradeView.py:39
#, python-format
msgid "%li seconds"
msgstr "%li sekúndur"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:56
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem"
msgstr ""
"Þetta niðurhal mun taka um %s á 1Mbit DSL tenginu og um %s á innhringi "
"tengingu"

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:60
#, fuzzy, python-format
msgid "This download will take about %s with your connection. "
msgstr ""
"Þetta niðurhal mun taka um %s á 1Mbit DSL tenginu og um %s á innhringi "
"tengingu "

#: ../DistUpgrade/DistUpgradeView.py:111 ../DistUpgrade/DistUpgrade.glade.h:14
#, fuzzy
msgid "Preparing to upgrade"
msgstr "Undirbý uppfærslu"

#: ../DistUpgrade/DistUpgradeView.py:112
#, fuzzy
msgid "Getting new software channels"
msgstr "Stilli hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgradeView.py:113 ../DistUpgrade/DistUpgrade.glade.h:12
#, fuzzy
msgid "Getting new packages"
msgstr "Villa í sóttu efni"

#: ../DistUpgrade/DistUpgradeView.py:114 ../DistUpgrade/DistUpgrade.glade.h:13
#, fuzzy
msgid "Installing the upgrades"
msgstr "Sæki og set upp uppfærslur"

#: ../DistUpgrade/DistUpgradeView.py:115 ../DistUpgrade/DistUpgrade.glade.h:8
msgid "Cleaning up"
msgstr "Hreinsa til"

#. FIXME: make those two seperate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:184
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakki mun vera fjarlægður."
msgstr[1] "%d pakkar munu vera fjarlægðir."

#: ../DistUpgrade/DistUpgradeView.py:189
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nýr pakki mun vera uppsettur."
msgstr[1] "%d nýir pakkar munu vera uppsettir."

#: ../DistUpgrade/DistUpgradeView.py:195
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakki mun vera uppfærður."
msgstr[1] "%d pakkar munu vera uppfærðir."

#: ../DistUpgrade/DistUpgradeView.py:200
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Þú þarft að niðurhala samtals %s. "

#: ../DistUpgrade/DistUpgradeView.py:205
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:211 ../UpdateManager/UpdateManager.py:692
msgid "Your system is up-to-date"
msgstr "Kerfið þitt er af nýjustu gerð"

#: ../DistUpgrade/DistUpgradeView.py:212
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Það eru engar uppfærslur fáanlega fyrir kerfið þitt. Uppfærslu verður nú hætt"

#: ../DistUpgrade/DistUpgradeView.py:225
msgid "Reboot required"
msgstr "Endurræsing nauðsynleg"

#: ../DistUpgrade/DistUpgradeView.py:226
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "Uppfærslu er lokið og endurræsing er nauðsynleg. Vilt þú gera svo nú?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:143
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:143
msgid "Could not run the upgrade tool"
msgstr "Gat ekki keyrt uppfærslu tólið"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:144
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:144
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Það er sennilega villa í uppfærslutólinu. Vinamelgast tilkynntu það sem "
"villu."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:170
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:170
msgid "Upgrade tool signature"
msgstr "Auðkenni uppfærslutóls"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:173
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:173
msgid "Upgrade tool"
msgstr "Uppfærslutól"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:198
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:198
msgid "Failed to fetch"
msgstr "Mistókst að sækja"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:199
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:199
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Það mistókst að sækja uppfærsluna. Það gæti verið vandamál með netið. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:203
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:203
msgid "Failed to extract"
msgstr "Mistókst að draga upplýsingar út"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:204
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Það mistókst að draga upplýsingar um uppfærlsuna út.  Það gæti verið "
"vandamál með netið eða með þjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:209
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:209
msgid "Verfication failed"
msgstr "Sannprófun mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:210
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:210
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Sannprófun á uppfærslunni mistókst. Það gæti verið vandamál með netið eða "
"með þjóninn. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:215
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:215
msgid "Authentication failed"
msgstr "Staðfesting aðkenna mistókst"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:216
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:216
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Það mistókst að staðfesta auðkenni uppfærslunar. Það gæti verið vandamál með "
"netið eða með þjóninn. "

#: ../DistUpgrade/DistUpgradeViewText.py:99
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade aborts "
"now.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Vinsamlega tilkynntu þetta sem villu og hafðu skrárnar /var/log/dist-upgrade/"
"main.log og /var/log/dist-upgrade/apt.log innifaldar í skýrslunni þinni. "
"Uppfærslan hættir vinnu núna.\n"
"Upprunalega sources.list skráin þín var vistuð í /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:123
msgid "Aborting"
msgstr "Hætti við..."

#: ../DistUpgrade/DistUpgradeViewText.py:128
#, fuzzy
msgid "Demoted:\n"
msgstr "_Fjarlægja\n"

#: ../DistUpgrade/DistUpgradeViewText.py:148
#: ../DistUpgrade/DistUpgradeViewText.py:172
#, fuzzy
msgid "Continue [yN] "
msgstr "_Halda áfram "

#: ../DistUpgrade/DistUpgradeViewText.py:148
#, fuzzy
msgid "Details [d]"
msgstr "Nánar"

#. TRANSLATORS: the "y" is "yes"
#: ../DistUpgrade/DistUpgradeViewText.py:152
#: ../DistUpgrade/DistUpgradeViewText.py:176
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#: ../DistUpgrade/DistUpgradeViewText.py:155
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:158
msgid "d"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:161
#, fuzzy, python-format
msgid "Remove: %s\n"
msgstr "_Fjarlægja\n"

#: ../DistUpgrade/DistUpgradeViewText.py:163
#, fuzzy, python-format
msgid "Install: %s\n"
msgstr "Uppsetning á %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:165
#, fuzzy, python-format
msgid "Upgrade: %s\n"
msgstr "Uppfæri %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:181
#, fuzzy
msgid "Restart required"
msgstr "Endurræsing nauðsynleg"

#: ../DistUpgrade/DistUpgradeViewText.py:182
#, fuzzy
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Til að klára uppfærsluna, þarfnast endurræsingar.\n"
"Ef þú velur 'j' mun kerfið endurræsa sig."

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, fuzzy
msgid "Restart the system now [yN] "
msgstr "Endurræsi kerfi "

#: ../DistUpgrade/DistUpgrade.glade.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.glade.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Endurræsa kerfið til að ljúka uppfærslunni</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Byrja á uppfærslunni?</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:7
msgid "<b><big>Upgrading Ubuntu to version 8.04 LTS</big></b>"
msgstr "<b><big>Uppfæri Ubuntu í útgáfu 8.04 LTS</big></b>"

#: ../DistUpgrade/DistUpgrade.glade.h:10
msgid "Difference between the files"
msgstr "Munur milli skránna"

#: ../DistUpgrade/DistUpgrade.glade.h:11
msgid "Distribution Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgrade.glade.h:15
#, fuzzy
msgid "Restarting the computer"
msgstr "Endurræsi kerfi"

#: ../DistUpgrade/DistUpgrade.glade.h:16
#, fuzzy
msgid "Setting new software channels"
msgstr "Stilli hugbúnaðarrásir"

#: ../DistUpgrade/DistUpgrade.glade.h:17
msgid "Terminal"
msgstr "Útstöð"

#: ../DistUpgrade/DistUpgrade.glade.h:18
msgid "_Cancel Upgrade"
msgstr "_hætta við"

#: ../DistUpgrade/DistUpgrade.glade.h:19
msgid "_Continue"
msgstr "_Halda áfram"

#: ../DistUpgrade/DistUpgrade.glade.h:21
msgid "_Replace"
msgstr "_Skipta út"

#: ../DistUpgrade/DistUpgrade.glade.h:22
msgid "_Report Bug"
msgstr "_Tilkynna Villu"

#: ../DistUpgrade/DistUpgrade.glade.h:23
msgid "_Restart Now"
msgstr "_Endurræsa núna"

#: ../DistUpgrade/DistUpgrade.glade.h:24
msgid "_Resume Upgrade"
msgstr "Hefja _aftur uppfærslu"

#: ../DistUpgrade/DistUpgrade.glade.h:25
msgid "_Start Upgrade"
msgstr "_Byrja uppfærslu"

#: ../UpdateManager/DistUpgradeFetcher.py:79
msgid "Could not find the release notes"
msgstr "Fann ekki upplýsingar um útgáfu"

#: ../UpdateManager/DistUpgradeFetcher.py:80
msgid "The server may be overloaded. "
msgstr "Þjóninn gæti verið með yfirálag "

#: ../UpdateManager/DistUpgradeFetcher.py:90
msgid "Could not download the release notes"
msgstr "Gat ekki halað niður upplýsingum um útgáfu"

#: ../UpdateManager/DistUpgradeFetcher.py:91
msgid "Please check your internet connection."
msgstr "Vinsamlegast athugaðu internettenginguna þína"

#: ../UpdateManager/GtkProgress.py:108
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""

#: ../UpdateManager/GtkProgress.py:113
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr ""

#: ../UpdateManager/UpdateManager.py:246
msgid "The list of changes is not available"
msgstr ""

#: ../UpdateManager/UpdateManager.py:252
msgid ""
"The list of changes is not available yet.\n"
"Please try again later."
msgstr ""

#: ../UpdateManager/UpdateManager.py:257
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""

#: ../UpdateManager/UpdateManager.py:284
msgid "Important security updates"
msgstr "Mikilvægar öryggisuppfærslur"

#: ../UpdateManager/UpdateManager.py:285
msgid "Recommended updates"
msgstr "Uppfærslur sem mælt er með"

#: ../UpdateManager/UpdateManager.py:286
msgid "Proposed updates"
msgstr "Uppástunga að uppfærslum"

#: ../UpdateManager/UpdateManager.py:287
msgid "Backports"
msgstr ""

#: ../UpdateManager/UpdateManager.py:288
msgid "Distribution updates"
msgstr ""

#: ../UpdateManager/UpdateManager.py:293
msgid "Other updates"
msgstr ""

#: ../UpdateManager/UpdateManager.py:519
#, python-format
msgid "Version %s: \n"
msgstr "Útgáfa %s: \n"

#: ../UpdateManager/UpdateManager.py:580
#, fuzzy
msgid "Downloading list of changes..."
msgstr "Hala niður breytingarlista..."

#: ../UpdateManager/UpdateManager.py:607
msgid "_Uncheck All"
msgstr ""

#: ../UpdateManager/UpdateManager.py:613
msgid "_Check All"
msgstr ""

#. TRANSLATORS: b stands for Bytes
#: ../UpdateManager/UpdateManager.py:657 ../UpdateManager/UpdateManager.py:709
#, python-format
msgid "Download size: %s"
msgstr "Stærð niðurhals: %s"

#: ../UpdateManager/UpdateManager.py:661
msgid "Unknown download size"
msgstr "Óþekkt stærð niðurhals"

#: ../UpdateManager/UpdateManager.py:669
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""

#: ../UpdateManager/UpdateManager.py:678
#, python-format
msgid "The package information was last updated %s days ago."
msgstr "Pakkaupplýsingarnar voru seinast uppfærðar fyrir %s dögum síðan."

#: ../UpdateManager/UpdateManager.py:680
#, fuzzy, python-format
msgid "The package information was last updated %s hours ago."
msgstr "Pakkaupplýsingarnar voru seinast uppfærðar fyrir %s klst síðan."

#: ../UpdateManager/UpdateManager.py:682
msgid "The package information was last updated less than one hour ago."
msgstr "Pakkaupplýsingarnar voru seinast uppfærðar fyrir minna en klst síðan."

#: ../UpdateManager/UpdateManager.py:690
#: ../data/glade/UpdateManager.glade.h:21
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Hugbúnaðauppfærslur leiðrétta villur, lagfæra öryggisveilur og innleiða nýja "
"möguleika."

#: ../UpdateManager/UpdateManager.py:705
#, fuzzy, python-format
msgid "You can install %s update."
msgid_plural "You can install %s updates."
msgstr[0] "Gat ekki sett upp uppfærslur"
msgstr[1] "Gat ekki sett upp uppfærslur"

#: ../UpdateManager/UpdateManager.py:746
msgid "Please wait, this can take some time."
msgstr "Vinsamlegast dokaðu, þetta gæti tekið drjúga stund."

#: ../UpdateManager/UpdateManager.py:748
msgid "Update is complete"
msgstr "Uppfærslu er lokið"

#: ../UpdateManager/UpdateManager.py:800
msgid "Reading package information"
msgstr "Les pakkaupplýsingar"

#: ../UpdateManager/UpdateManager.py:877
msgid "Could not initialize the package information"
msgstr ""

#: ../UpdateManager/UpdateManager.py:878
msgid ""
"A unresolvable problem occurred while initializing the package information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""

#: ../UpdateManager/UpdateManager.py:900
msgid ""
"A unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:925
#, python-format
msgid "(Size: %s)"
msgstr "(Stærð: %s)"

#: ../UpdateManager/UpdateManager.py:935
msgid "Your distribution is not supported anymore"
msgstr "Dreifingin þin er ekki studd lengur"

#: ../UpdateManager/UpdateManager.py:936
#, fuzzy
msgid ""
"You will not get any further security fixes or critical updates. Upgrade to "
"a later version of Ubuntu Linux. See http://www.ubuntu.com for more "
"information on upgrading."
msgstr ""
"Þú færð ekki fleiri öryggislagfæringar eða mikilvægar uppfærslur. Þú ættir "
"að uppfæra í nýrri útgáfu af  Ubuntu Linux. Sjá http://www.ubuntu.com fyrir "
"nánari upplýsingar um uppfærslur."

#: ../UpdateManager/UpdateManager.py:951
msgid "Downloading the upgrade tool"
msgstr "Hala niður uppfærslutólinu"

#: ../UpdateManager/UpdateManager.py:953
msgid "The upgrade tool will guide you through the upgrade process."
msgstr "Uppfærslutólið mun hjálpa þér í gegnum uppfærlsuferlið."

#: ../UpdateManager/UpdateManager.py:961
#, python-format
msgid "<b>New distribution release '%s' is available</b>"
msgstr "<b>Ný útgáfa dreifngarinnar ,%s' er tiltæk</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1004
msgid "Software index is broken"
msgstr "Hugbúnaðarvísir er bilaður"

#: ../UpdateManager/UpdateManager.py:1005
#, fuzzy
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Það er ómögulegt að setja upp eða fjarlægja nokkurn hugbúnað. Vinsamlegast "
"notaðu pakkastjórann „Synaptic‟ eða keyrðu skipunina „sudo apt-get install -"
"f‟ í skjáhermi til að laga þetta vandamál fyrst."

#: ../data/glade/UpdateManager.glade.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""

#: ../data/glade/UpdateManager.glade.h:4
#, fuzzy
msgid "<big><b>Keep your system up-to-date</b></big>"
msgstr "<big><b>Haltu kerfinu upfærðuu</b></big>"

#: ../data/glade/UpdateManager.glade.h:5
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "d"

#: ../data/glade/UpdateManager.glade.h:7
msgid "Changes"
msgstr "Breytingar"

#: ../data/glade/UpdateManager.glade.h:8
msgid "Chec_k"
msgstr "_Kanna"

#: ../data/glade/UpdateManager.glade.h:9
msgid "Check the software channels for new updates"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:10
msgid "Description"
msgstr "Lýsing"

#: ../data/glade/UpdateManager.glade.h:11
msgid "Description of update"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:12
msgid "Release Notes"
msgstr "Útgáfu upplýsingar"

#: ../data/glade/UpdateManager.glade.h:13
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" ◦ A previous upgrade which didn't complete\n"
" ◦ Problems with some of the installed software\n"
" ◦ Unofficial software packages not provided by Ubuntu\n"
" ◦ Normal changes of a pre-release version of Ubuntu"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:20
msgid "Show progress of individual files"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:22
msgid "U_pgrade"
msgstr "U_ppfæra"

#: ../data/glade/UpdateManager.glade.h:23
#: ../data/update-manager.desktop.in.h:3
msgid "Update Manager"
msgstr "Uppfærslustjóri"

#: ../data/glade/UpdateManager.glade.h:24
msgid "Upgrade to the latest version of Ubuntu"
msgstr "Uppfæra í nýjustu útgáfu af Ubuntu"

#: ../data/glade/UpdateManager.glade.h:25
msgid "_Check"
msgstr "_Athuga"

#: ../data/glade/UpdateManager.glade.h:26
msgid "_Hide this information in the future"
msgstr "_Fela þessar upplýsingar í framtíðinni"

#: ../data/glade/UpdateManager.glade.h:27
msgid "_Install Updates"
msgstr "_Setja upp uppfærslur"

#: ../data/glade/UpdateManager.glade.h:28
msgid "_Partial Upgrade"
msgstr ""

#: ../data/glade/UpdateManager.glade.h:29
msgid "_Upgrade"
msgstr "_Uppfæra"

#: ../data/glade/UpdateManager.glade.h:30
msgid "updates"
msgstr ""

#: ../data/update-manager.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Sýna og setja upp tiltækar dagréttingar"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr ""

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr ""

#: ../data/update-manager.schemas.in.h:2
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""

#: ../data/update-manager.schemas.in.h:3
msgid "Remind to reload the channel list"
msgstr "Minna á að endurhlaða rásalista"

#: ../data/update-manager.schemas.in.h:4
msgid "Show details of an update"
msgstr ""

#: ../data/update-manager.schemas.in.h:5
msgid "Stores the size of the update-manager dialog"
msgstr ""

#: ../data/update-manager.schemas.in.h:6
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""

#: ../data/update-manager.schemas.in.h:7
msgid "The window size"
msgstr "Gluggastærð"

#: ../data/update-manager.schemas.in.h:8
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""

#: ../update-manager:54
msgid "Show version and exit"
msgstr "Sýna útgáfu og hætta"

#: ../update-manager:57
#, fuzzy
msgid "Check if a new distribution release is available"
msgstr "Athuga hvort ný dreifing sé til staðar"

#: ../update-manager:60 ../do-release-upgrade:19
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""

#: ../update-manager:64 ../update-manager:67
msgid "Try to run a dist-upgrade"
msgstr "Reyndu að keyra drefingaruppfærslu"

#: ../update-manager:85
msgid "Running partial upgrade"
msgstr ""

#: ../update-manager:105
msgid "Upgrade complete"
msgstr "Uppfærslu lokið"

#: ../update-manager:106
msgid "The upgrade was completed."
msgstr "Uppfærslunni var lokið."

#: ../do-release-upgrade:23
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""

#: ../do-release-upgrade:27
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""

#: ../do-release-upgrade:33
msgid "Run the specified frontend"
msgstr ""

#: ../do-release-upgrade:37
msgid "Checking for a new ubuntu release"
msgstr ""

#: ../do-release-upgrade:44
msgid "No new release found"
msgstr "Ekki fundust nýjar útgáfur"

#, fuzzy
#~ msgid "Fetching the upgrades"
#~ msgstr "Undirbý uppfærslu"

#~ msgid "_Skip This Step"
#~ msgstr "_Sleppa þessu skrefi"
